app/models/concerns/has_scrubbable_fields.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module HasScrubbableFields
```

Comment:
Any chance we can reuse `HasSanitizableFields`?

Reply:
we could - yes. I thought this was cleaner because HasSanitizableFields would become a bit hacky and confusing. 

that is, you could do it by adding a `tags` parameter to `sanitize_field`, but it would have to work in the opposite sense of the `attributes` param. that is, it would need to *replace* the set of allowed tags as opposed to adding to it - because we don't want other html tags in tag descriptions (I am presuming).

let me know what you think.